src/components/WalletView/WalletView.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  Badge,
+  Box,
+  Flex,
+  Heading,
+  Text,
+  useColorModeValue
+} from '@chakra-ui/react';
+
+import { i18n } from '@/i18n';
+
+export const WalletView = (props) => {
+  const bgMain = useColorModeValue('orange.50', 'teal.800');
+  const textColor = useColorModeValue('teal.900', 'orange.100');
+  const bgBadge = useColorModeValue('orange.100', 'teal.700');
+
+  const totalBalanceView = new Intl.NumberFormat('de-DE', {
+    minimumFractionDigits: 2
+  }).format(props.totalBalance);
+
+  return (
+    <>
+      <Box h="200px" w="400px" m={5} borderRadius={35} bg={bgMain} shadow="lg">
+        <Badge
+          ml="80%"
+          mt="5%"
+          fontSize="50%"
+          align="center"
+          p={1}
+          bg={bgBadge}
+          borderRadius={5}
+          color={textColor}
+        >
+          default
```

Comment:
i18n
Got you :)

Reply:
but I'm not sure that we should translate it.

Reply:
done